### DISCUSSION TITLE
OS License for the Beman Project

### DISCUSSION THREAD

**Discussion by inbal2l** [bemanproject/beman#9] [General]
Recommendations to license (by Inbal Levi, as representative of "Hamakor" non-profit).
As presented in C++Now 2024 (with minor fixes).

[Beman project - OS Licence.pdf](https://github.com/beman-org/beman/files/15191130/Beman.project.-.OS.Licence.pdf)

Recommendation for the "root":
- No license, or
- The Unlicense, or
- MIT

(to be determined by attendees)

**Comment by JeffGarland** [comment]
Note that we've changed the default to MIT for the moment

**Reply by inbal2l** [reply]
Great, thanks for the update!
Closing issue for now, let's reopen if reconsideration is needed / new information comes by.
